kernel/static/loader/empty-scenes-halloween/index.json

```diff
@@ -0,0 +1,413 @@
+{
+  "EP_01": [
+    {
+      "file": "bin/game.js",
+      "hash": "EP_01.js"
+    },
+    {
+      "file": "scene.json",
+      "hash": "scene.json"
+    },
+    {
+      "file": "models/FloorBaseGrass_01/FloorBaseGrass_01.glb",
+      "hash": "QmSyvWnb5nKCaGHw9oHLSkwywvS5NYpj6vgb8L121kWveS"
+    },
+    {
+      "file": "models/FloorBaseGrass_01/Floor_Grass01.png.png",
+      "hash": "QmT1WfQPMBVhgwyxV5SfcfWivZ6hqMCT74nxdKXwyZBiXb"
+    },
+    {
+      "file": "models/FloorBaseGrass_01/thumbnail.png",
+      "hash": "QmexuPHcbEtQCR11dPXxKZmRjGuY4iTooPJYfST7hW71DE"
+    },
+    {
+      "file": "models/EP_01.glb",
+      "hash": "QmHalloween01"
+    },
+    {
+      "file": "models/emptyparcels-tx.png",
+      "hash": "QmHalloweenEmptyParcels"
```

Comment:
As you can see, the Qm hashes are fruta. We should get the proper hashes before submitting those assets to content server/asset bundles s3.